frame/support/test/tests/derive_no_bound.rs

```diff
@@ -0,0 +1,177 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Tests for DebugNoBound, CloneNoBound, EqNoBound, PartialEqNoBound, and RuntimeDebugNoBound
+
+use frame_support::{DebugNoBound, CloneNoBound, EqNoBound, PartialEqNoBound, RuntimeDebugNoBound};
+
+#[derive(RuntimeDebugNoBound)]
+struct Unnamed(u64);
+
+#[test]
+fn runtime_debug_no_bound_display_correctly() {
+	#[cfg(feature = "std")]
+	{
+		assert_eq!(format!("{:?}", Unnamed(1)), "Unnamed(1)");
+	}
+
+	#[cfg(not(feature = "std"))]
+	{
+		assert_eq!(format!("{:?}", Unnamed(1)), "<stripped>");
```

Comment:
Is this ever tested? :P

Reply:
indeed it isn't, I removed the confusing dead code